Makefile

```diff
@@ -23,6 +23,7 @@ stop:
 	make clean
 
 clean:
+	[ "$$(cat subnet_assignment.state)" = "" ] || rm -rf /tmp/volplugin_vagrant_subnets/`cat subnet_assignment.state`  || true
```

Comment:
* Use `-n` negated to test the empty string
* Instead of using `true`, use `:`.